share/lib/python/neuron/rxd/region.py

```diff
@@ -510,15 +515,16 @@ def __init__(self, secs=None, nrn_region=None, geometry=None, dimension=None, dx
         """
         self._allow_setting = True
         if hasattr(secs,'__len__'):
-            self.secs = secs
+            self._secs = secs
         else:
```

Comment:
Should we add a `secs` property that returns a Python list (of, I guess, the sections in `self._secs`) so that we're not taking things away from the public interface?